depthai_sdk/setup.py

```diff
@@ -5,12 +5,11 @@
 with open('requirements.txt') as f:
     requirements = f.readlines()
 
-install_requires=[requirement for requirement in requirements if '--' not in requirement]
-
+install_requires = [requirement for requirement in requirements if '--' not in requirement]
 
 setup(
     name='depthai-sdk',
-    version='1.9.3',
+    version='1.9.2.1',
```

Comment:
1.9.3 to 1.9.2.1?

Reply:
The last released version was 1.9.2, so I think this one can be 1.9.2.1. Next bigger release would be 1.9.3 then. What do you think?

Reply:
I think we can use the 4th number for bugfixes only - so for small releases we change 3rd num, and for big ones we can `num++` 2nd one. Thoughts?

Reply:
Yeah, totally agree. Changing this to 1.9.3, next one will be 1.9.4👌